### PR TITLE
Clarify new privilege requirements for SHOW commands.

### PR DESCRIPTION
**Pull request by Lojjs**
Based on the new rules introduced in https://github.com/neo-technology/neo4j/pull/5302

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 1]
```diff
@@ -361,7 +361,8 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
 
-        p("The ability to show roles can be granted via the `SHOW ROLE` privilege. The following query shows an example of this:")
+        p("The ability to show roles can be granted via the `SHOW ROLE` privilege. A user with this privilege is allowed to execute the `SHOW ROLES` and `SHOW POPULATED ROLES` administration commands. " +
+          "For the `SHOW ROLES WITH USERS` and `SHOW POPULATED ROLES WITH USERS` administration command, both this privilege and the `SHOW USER` privilege are required. The following query shows an example of how to grant the `SHOW ROLE` privilege:")
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]:
```suggestion
          "For the `SHOW ROLES WITH USERS` and `SHOW POPULATED ROLES WITH USERS` administration commands, both this privilege and the `SHOW USER` privilege are required. The following query shows an example of how to grant the `SHOW ROLE` privilege:")
```

**Review by SaschaPeukert — CHANGES_REQUESTED** [review]
Seems good to me, except for one suggestion

**Review by SaschaPeukert — APPROVED** [review]

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 2]
```diff
@@ -508,6 +510,10 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
 
+        note {
+          p("Note that no specific privileges are required for showing the current user's privileges using `SHOW USER [username] PRIVILEGES`.")
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: In line 501 we use `SHOW USER username PRIVILEGES`, and in this line we use `SHOW USER [username] PRIVILEGES` - it seems that the first one is correct?

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @Lojjs and @SaschaPeukert 

One small question - happy to fix this myself if that's helpful?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala (hunk 2 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: `SHOW USER username PRIVILEGES`is the usual command but in 4.1 you can also type `SHOW USER PRIVILEGES` without any username to get your own privileges. What I tried to write here is that you can run `SHOW USER PRIVILEGES` as well as `SHOW USER username PRIVILEGES` with your own username without any specific privilege. 

If you find any clearer way of formulating that, please go ahead

Comment (davidoliverSP2) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: thanks @Lojjs - I pushed a small update to add that users can run `SHOW USER username PRIVILEGES` or `SHOW USER PRIVILEGES` for showing the current user's privileges

Comment (Lojjs) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: Thanks looks good

**Pull request merged**